samples/nrf9160/aws_fota/README.rst

```diff
@@ -29,6 +31,55 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+Prerequisites
+*************
+
+The connected device must have the same MQTT Client ID as the thing name created on AWS IoT.
+The device needs to have certificates from AWS IoT provisioned onto the device and it needs to have a policy attached to the certificates, and that the certificates are activate in AWS IoT. The process for doing so is described below.
+
+Below is an example of a very permissive policy which should be used for **testing only**.
+You can attach it to the certificates if you are unsure what policies are needed for your device.
+To create a policy, go to `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and select **Secure->Policies->Create**.
+
+.. code-block:: javascript
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": "iot:*",
+            "Resource": "*"
+          }
+       ]
+     }
+
+To create a thing, log in to the `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and select **Manage->Things->Create->Create a single thing**.
+Provide it with a name which corresponds to your device's MQTT Client ID.
+Next, choose to either to create certificates or add your own certificates and make sure to **Activate** them.
+Also make sure to download one of the `root CA certificates <https://docs.aws.amazon.com/iot/latest/developerguide/managing-device-certs.html#server-authentication>`_ and the generated certificates if you chose to let AWS generate some for layou.
+
+Then, attach the policy you created earlier to the certificates. Further more you need to the information from the Root CA, Public Certificate, and Private Key into :file:`certificates.h`.
+
+When these steps are done, deselect ``CONFIG_USE_PROVISIONED_CERTIFICATES`` in menuconfig.
+This will write your new certificates to the :ref:`secure tag <zephyr:sec_tag_t>` defined in menuconfig ``CONFIG_CLOUD_CERT_SEC_TAG`` into the modem.
+
+
+Usage
+*****
+
+1. Make sure you have configured the host name to be the AWS IoT MQTT Broker endpoint you want to use in menuconfig.
+#. If you haven't already provisioned certificates on to your device follow the steps in :ref:`prerequisites`.
+#. Flash the sample onto the board.
+#. When the sample has successfully connected to the AWS IoT broker, log in to your `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and go to **Manage->Jobs->Create**.
+#. Then, choose **Create custom job**, provide it with a unique **Job ID**, select the **Device to update**, and add a **Job file**.
```

Comment:
Try to split this step further. If you get "add a Job file" as a separate point, add the note below (about the example job file) to that point since I think many users will not read everything before trying to follow the steps :).

Reply:
okei :)

Reply:
forgot to push this yesterday it seems. Does it looks okay from your perspective now @b-gent ?